### DISCUSSION TITLE
Ошибка Для сравнения доступны только

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#188] [Как это сделать?] [answered]
![image](https://user-images.githubusercontent.com/103171127/183929758-85042b98-c917-44c5-a038-5cb8dfd9db33.png)
При вывода данных с помощью insert выдаёт ошибку "Для сравнения доступны только =, ==, !=, !==" ... 
С чем это связано?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Весь затык в том, что в этом случае в поле insert передается `row` — это видно по кусочку содержимого в `h_insert`. Также для поле `insert` выбрано полем сортировки. Сортировка это сравнение (там далеко внутри модуль тот-же, что и для сравнений) — а сравнения `>` `<` со списками и `row` не проходят.

**Поэтому немного неочевидный текст ошибки.**

В этом случае, что бы исправить, надо изменить в настройках таблицы поле сортировки.
